CONTRIBUTING.md

```diff
@@ -1,128 +1,82 @@
-
-### Late-Stage Process
-
-The late-stage process attempts to reflect the Working Group's current consensus in the drafts; the latest draft reflects that consensus, modulo any open (or undiscovered) issues. The goal for a late-stage draft is to reduce unnecessary design changes in the protocol, thereby aiding reviewers and assuring that the drafts accurately reflect consensus.
-
-In this process, the Working Group will discuss each design issue, and the Chairs will judge consensus, labelling the issue as `has-consensus` (ideally based upon a Pull Request that specifies the exact changes to be made).
-
-Only after that will the change be merged and the issue be closed.
-
-The drafts currently in the late stage are:
-
-* Invariants
-* Transport
-* TLS
-* HTTP/3
-* QPACK
-* Recovery
-
-![diagram of the late stage workflow](workflow.png "Late Stage Workflow")
-
-### Discretionary Design Issue Labels
-
-We also use the following labels to help understand the state of our design issues:
-
-* [`arch`](https://github.com/quicwg/base-drafts/labels/arch): The issue is a higher-level architectural issue that should drive the solution to a number of other issues.
-* [`needs-discussion`](https://github.com/quicwg/base-drafts/labels/needs-discussion): The issue blocks progress to our next milestone.
-* [`editor-ready`](https://github.com/quicwg/base-drafts/labels/editor-ready): The Working Group believes it has a viable resolution, but the editors need to incorporate that into the document so we can see it in situ.
-
-
-## Pull Requests
-
-We welcome pull requests, both for editorial suggestions and to resolve open issues. In the latter
-case, please identify the relevant issue.
+## Following Discussion
 
-Please do not use a pull request to open a new design issue; it may not be noticed.
+The Working Group has a few venues for discussion:
 
+* We plan to meet at all [IETF meetings](https://www.ietf.org/meeting/) for the
+  foreseeable future, and possibly hold interim meetings between them as
+  required. Agendas, minutes, and presentations are available in our [meeting
+  materials repository](https://github.com/quicwg/wg-materials) and the
+  [official proceedings](https://datatracker.ietf.org/wg/quic/meetings/).
+
+* Our [mailing list](https://www.ietf.org/mailman/listinfo/quic) is used for
+  most communication, including notifications of meetings, new drafts, consensus
+  calls and other business, as well as issue discussion.
+
+* We maintain several repositories in our GitHub organization
+  [Github](https://github.com/quicwg/). Specific issues are discussed on the
+  relevant issues list. If you don't want to use Github to follow these
+  discussions, you can subscribe to the [issue announce
+  list](https://www.ietf.org/mailman/listinfo/quic-issues).
+
+To be active in the Working Group, you can participate in any of these places.
+Most activity takes place on the mailing list, but if you just want to comment
+on and raise issues, that's fine too.
 
```

Comment:
I'll note that we're not mentioning the Slack channel here - should we?

Reply:
good idea, done.